scripts/catalog-tests/generate-tdr-snapshot-metadata.py

```diff
@@ -0,0 +1,127 @@
+#------------------------------------------------------------------------------
+# Generate TDR Snapshot Metadata
+#
+# This script adds snapshots to the catalog as the test user from DC-218,
+# pulling the metadata from the resources JSON file
+# 
+# Before Running:
+#   python3 -m venv .env
+#   source .env/bin/activate
+#   pip install --upgrade pip
+#   python3 -m pip install requests
+#   
+#   Authorization: Access the password in the vault using the appropriate command:
+#   vault login -method=github token=$(cat ~/.github-token)
+#   vault read secret/dsde/terra/kernel/dev/dev/catalog/tests/user
+#   vault read secret/dsde/terra/kernel/dev/dev/catalog/tests/userAdmin
+# Run: python3 generate-tdr-snapshot-metadata.py
```

Comment:
Maybe give an example using `catalogServiceUrl` so the user knows how to configure the tool.